pkg/symbols/symbols.go

```diff
@@ -0,0 +1,190 @@
+package symbols
+
+import (
+	"context"
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/sourcegraph/go-lsp"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/backend"
+	"github.com/sourcegraph/sourcegraph/pkg/api"
+	"github.com/sourcegraph/sourcegraph/pkg/gituri"
+	"github.com/sourcegraph/sourcegraph/pkg/symbols/protocol"
+	"gopkg.in/inconshreveable/log15.v2"
+)
+
+type ComputeSymbolsArgs struct {
```

Comment:
Elsewhere in code, we use the suffix `Op` for option struct names.
```suggestion
type ComputeSymbolsOp struct {
```